instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py

```diff
@@ -24,6 +24,7 @@
 
 import opentelemetry.instrumentation.elasticsearch
 from opentelemetry import trace
+from opentelemetry import trace as trace_api
```

Comment:
trace already imported in the upper line

Reply:
Done